Makefile

```
@@ -107,6 +107,26 @@ vendor:
 	# Nuke vendor directory
 	rm -rf vendor
 
+	# rebuild dependencies
+	dep ensure -v
```

instead of a separate `vendor-update` script, maybe we could just do:

```suggestion
	dep ensure -v $(DEPFLAGS)
```

then:

```
make vendor DEPFLAGS=-update
```

---

neat!